packages/web-components/src/components/data-table/table-batch-actions.ts

```diff
@@ -30,6 +30,17 @@ class CDSTableBatchActions extends LitElement {
     );
   }
 
+  /**
+   * Handles `click` event on the Select all button.
+   */
+  private _handleSelectAll() {
+    const { eventClickSelectAll } = this
+      .constructor as typeof CDSTableBatchActions;
+    this.dispatchEvent(
+      new CustomEvent(eventClickSelectAll, { bubbles: true, composed: true })
+    );
+  }
```

Comment:
```suggestion
  private _handleSelectAll = () => {
  this.dispatchEvent(
    new CustomEvent(CDSTableBatchActions.eventClickSelectAll, {
      bubbles: true,
      composed: true,
    })
  );
};

```

I don't think you need to destructure here, you can access directly.